### PR TITLE
fix figshare links

### PR DESCRIPTION
**Pull request by cxhernandez**
re: https://github.com/msmbuilder/msmbuilder/issues/750

### CODE REVIEW

**Comment by mpharrigan** [comment]
Well that's annoying...

**Comment by cxhernandez** [comment]
Yeah... I'll look into why these files aren't downloading correctly

**Comment by cxhernandez** [comment]
@mpharrigan.... https://travis-ci.org/msmbuilder/msmbuilder/jobs/114950676#L2035

**Comment by mpharrigan** [comment]
the example notebooks are failing to run

**Comment by mpharrigan** [comment]
probably because they lean heavily on the figshare data

File: msmbuilder/tests/test_commands.py [hunk 1]
```diff
@@ -149,7 +149,7 @@ def test_superpose_featurizer_reftop():
         shell("msmb SuperposeFeaturizer "
               "--trjs '{data_home}/alanine_dipeptide/*.dcd'"
               " --transformed distances --atom_indices all.txt"
-              " --reference_traj {data_home}/alanine_dipeptide/trajectory_0.dcd"
+              " --reference_traj {data_home}/alanine_dipeptide/trajectory-0.dcd"
```
Comment (mpharrigan) [inline msmbuilder/tests/test_commands.py]: Why did these change??

Comment (cxhernandez) [inline msmbuilder/tests/test_commands.py]: I think figshare changed the naming scheme somehow, replacing underscores with hyphens.

**Comment by cxhernandez** [comment]
woo!

**Comment by mpharrigan** [comment]
how did figshare change the filenames?

**Comment by cxhernandez** [comment]
I have no clue. It was actually only `fs_peptide.pdb` that got changed: https://figshare.com/articles/Fs_MD_Trajectories/1030363.

**Pull request merged**